src/components/switch/switch.scss

```
@@ -1,6 +1,6 @@
+@forward './partial-styles/_helper-text.scss';
```

Stylelint (17.3.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)